src/Components/SourcesSelect/index.js

```diff
@@ -24,9 +24,18 @@ const SourcesSelect = ({ setValidation }) => {
     enabled: !!provider,
     onSuccess: (data) => {
       const id = chosenSource;
-
-      if (!id) return;
-      setSelected(selectObject(id, data.find((source) => source.id === id).name));
+      if (provider == AWS_PROVIDER) {
```

Comment:
should we limit this for only the cases the source ID was passed?
```suggestion
      if (provider == AWS_PROVIDER && imageSourceID) {
```
That way we would allow for the provisioning of images shared with account.

1. it would allow to keep the simple pipeline we have currently (testing is easier)
2. if we ever decide to allow for the account shared images, we can do so easily :)

Reply:
sure, I don't have a problem just thought we are not enabling it. 
But maybe it will be smarter to start this way and change in the future if we see fit.